cli/src/commands/enrich.rs

```diff
@@ -1,7 +1,14 @@
 use std::str::FromStr;
```

Comment:
@qtpeters  Should `enrich` either 

1. Be replaced with an alternate subcommand called `findings`
OR
2. Have an a additional subcommand called `findings`
OR
3. Have flay called `--kind findings` ?